web/rest/platform/features/provider/mediaRelaySet/removeMediaRelaySet.feature

```diff
@@ -10,3 +10,19 @@ Feature: Manage media relay sets
       And I add "Accept" header equal to "application/json"
       And I send a "DELETE" request to "/media_relay_sets/1"
      Then the response status code should be 204
+
+  @createSchema
+  Scenario: Remove default media relay sets
+    Given I add Authorization header
+     When I add "Content-Type" header equal to "application/json"
+      And I add "Accept" header equal to "application/json"
+      And I send a "DELETE" request to "/media_relay_sets/0"
+     Then the response status code should be 403
+      And the response should be in JSON
+      And the header "Content-Type" should be equal to "application/problem+json; charset=utf-8"
+      And the JSON should be like:
+      """
+      {
+        "detail": "Rejected request during security check"
+      }
+      """
```

Comment:
I think its enough with the status code 403, this error comes from library I think, that's the reason you use should be like and not should be equal, but its up to you

Reply:
As a general rule I want to match both the failure code and the detail. The code is not enough because it can result in a test that passes because it fails as a result of another check: for instance, because the default media relay set is being used.